File: domain/Reaccionables/Habilidad.py
from datetime import datetime
from typing import List

from domain.Reaccionables.Expertise import Expertise
from domain.Reaccionables.RasgoValidable import RasgoValidable
from domain.Reaccionables.Reaccion import Reaccion


class Habilidad(RasgoValidable):
    def __init__(self, reacciones: List[Reaccion], texto: str, fecha_modificacion: datetime, expertise: Expertise) -> None:
        super().__init__(reacciones, texto, fecha_modificacion, expertise)

File: src/main/java/entities/usuarios/reaccionables/rasgos/Habilidad.java
package entities.usuarios.reaccionables.rasgos;

import entities.usuarios.reaccionables.Reaccion;

import javax.persistence.Entity;
import java.time.LocalDateTime;
import java.util.List;
@Entity
public class Habilidad extends RasgoValidable {

    public Habilidad(Topico topico, Expertise expertise, LocalDateTime fechaModificacion, List<Reaccion> reacciones) {
        super(topico, expertise, fechaModificacion, reacciones);
    }

    public Habilidad(){}
}
